src/main/resources/com/sap/oss/phosphor/fosstars/model/score/oss/OssSecurityScoreWeights.yml

```diff
@@ -0,0 +1,26 @@
+---
+values:
+  com.sap.oss.phosphor.fosstars.model.score.oss.ProjectSecurityAwarenessScore:
+    type: ImmutableWeight
+    value: 0.9
+  com.sap.oss.phosphor.fosstars.model.score.oss.UnpatchedVulnerabilitiesScore:
+    type: ImmutableWeight
+    value: 0.5 # unfortunately, we don't have a reliable source of info about unpatched vulnerabilities
+  com.sap.oss.phosphor.fosstars.model.score.oss.ProjectPopularityScore:
+    type: ImmutableWeight
+    value: 0.5
+  com.sap.oss.phosphor.fosstars.model.score.oss.ProjectActivityScore:
+    type: ImmutableWeight
+    value: 0.5
```

Comment:
I think these scores can have better weights
- The reason being, the data which can be actually crawled are used to calculate these scores. Yes? 
- Shouldn't the weight justify depending on its availability?

Reply:
I am not sure that I understand these two points. How exactly you would you update the weights?

Reply:
Well, my understanding is this:
- If there are few Data Providers which we are almost absolutely sure that we can get some data. May the scores generated using that data should have a higher weight.
- For example. we know we can get no of contributors of a github project, but LGTM, CodeQL related data providers may have no data.

Reply:
I'd use such knowledge to reduce weight if we know that data is not very reliable or complete. But possible upper bounds should not be derived from that. If we tune the security score, I believe we should think first what kind of scores tell about project's security the most. For example, if we have more reliable data about project activity, its weight should not be higher than the weight for security testing.